very-strict-eslint.json

```diff
@@ -0,0 +1,49 @@
+{
+  "root": true,
+  "env": {
+    "node": true
+  },
+  "extends": [
+    "plugin:vue/vue3-recommended",
+    "plugin:prettier-vue/recommended"
```

Comment:
is 'eslint:recommended' omitted intentionally?

Reply:
No you're right it should be there. I thought it was part of the other 2 but I see [this](https://eslint.vuejs.org/user-guide/#configuration) in the docs.